features/step_wrapper_with_options.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Step Wrapper with Options
+  In order to be able to write more complex step definition wrappers
+  As a developer
+  I want Cucumber to provide the "options" object to the wrapping function
+
+  Background:
+    Given a file named "features/a.feature" with:
+      """
+      Feature: Step with an option
+        Scenario: Steps
+          When I run a step with options
+      """
+    And a file named "features/step_definitions/cucumber_steps.js" with:
+      """
+      import {defineSupportCode} from 'cucumber'
+
+      defineSupportCode(({Then, When}) => {
+        When(/^I run a step with options$/, {retry: 2}, function () {
```

Comment:
Looks like we have a decision. Not the best for us, but hey, can't have everything ;) 
tests seem to be passing, looking forward to the new release so we can move forward with the wdio framework :)
Thanks!